dali/pipeline/executor/workspace_policy.h

```diff
@@ -390,14 +408,15 @@ struct AOT_WS_Policy<SeparateQueuePolicy> {
   template <OpType op_type, OpType group_as = op_type, typename T>
   void PlaceWorkspace(T &workspaces, QueueIdxs idxs, const OpGraph &graph,
                       const std::vector<tensor_data_store_queue_t> &tensor_to_store_queue,
-                      cudaStream_t mixed_op_stream, cudaStream_t gpu_op_stream,
+                      ThreadPool *thread_pool, cudaStream_t mixed_op_stream,
+                      cudaStream_t gpu_op_stream,
                       const std::vector<std::vector<cudaEvent_t>> &mixed_op_events) {
     int sequential_ws_idx = SequentialIndex(idxs, depths_, group_as);
     workspaces[sequential_ws_idx].resize(graph.NumOp(op_type));
     for (OpPartitionId partition_idx = 0; partition_idx < graph.NumOp(op_type); partition_idx++) {
       auto &node = graph.Node(op_type, partition_idx);
       workspaces[sequential_ws_idx][partition_idx] =
-          CreateWorkspace<op_type>(graph, node, tensor_to_store_queue, mixed_op_stream,
+          CreateWorkspace<op_type>(graph, node, tensor_to_store_queue, thread_pool, mixed_op_stream,
```

Comment:
See above: I'd rather see the two streams next to each other.

Reply:
done